.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,282 @@
+   ```bash
+   origin  https://github.com/<your_username>/incubator.git (fetch)
+   origin  https://github.com/<your_username>/incubator.git (push)
+   upstream  https://github.com/thegooddocsproject/incubator (fetch)
+   upstream  https://github.com/thegooddocsproject/incubator (push)
+   ```
+
+Now that you have your repository set up, you can create a branch to work on.
+
+#### Creating a branch
+
+1. In your terminal, navigate to the directory where you checked out the code:
+
+   ```bash
+   > cd ~/workspace/incubator
+   ```
+
+1. Fetch the branches in the upstream repository:
+
+   ```bash
+   > git fetch upstream
+   ```
+
+1. Check out your fork's `master` branch:
+
+   ```bash
+   > git checkout master
+   ```
+
+   This will show a message like this:
+
+   ```bash
+   Switched to branch `master`
+   ```
+
+1. Merge the changes from the upstream `master` branch, into your fork's `master` branch:
+
+   ```bash
+   > git merge upstream/master
+   ```
+
+   This shows some output that indicates what changes have merged in to your fork.
+1. Create a new branch for the work you want to do.
+   Make sure you give it an appropriate name, and include your username:
+
+   ```bash
+   > git checkout -b update-readme-username
+   ```
+
+Make your changes.
+When you have finished, save your changes, and create a pull request.
+
+Each branch and pull request must contain a single piece of work.
+For example, if you want to update a particular page and fix some typos across a lot of different pages, create a branch for the page update, make the changes, and do a pull request.
+Then create a new branch for the typos, and do a second pull request.
+This makes it easier for you to track your own work, and also for reviewers looking at your pull requests.
+
+#### Creating a pull request (PR)
+
+1. Save your changes in your text editor.
+1. In your terminal, add the file that contains your changes to your commit:
+
+   ```bash
+   > git add README.md
+   ```
+
+   If you have changes in a number of files, and you want to add all of them, you can use this command instead:
+
+   ```bash
+   > git add .
+   ```
+
+   Don't forget the period at the end of this command!
+1. Commit the changes, and provide a short description of what's in the commit:
+
+   ```bash
+   > git commit -m "Updated README file"
+   ```
+
+1. Push the changes to your fork:
+
+   ```bash
+   git push origin
+   ```
+
+1. In your browser, go to your fork on github.
+   The URL is in the form `github.com/<your_username>/incubator`, or you can get to it by clicking on your user image from the main github page, and selecting `Your repositories`.
+   In the banner at the top of the page, click `Compare and Create Pull Request`.
+1. Give your PR a title, and write a description of the changes.
+   Link any relevant issues, and tag people you want to review your changes.
+   If you intend to add more commits to your PR, mark your PR as a draft so that reviewers know you have not finished.
+
```

Comment:
**Chore:** After a night's sleep on this, I had the thought that we should clarify our policies about what requirements are necessary for a PR (who should review, how many approvals are needed, who is authorized to merge) and perhaps offer some guidelines about getting a PR reviewed. Every open source project has their own culture and rules around PRs and so it's something we need to clarify. I think that's a discussion that is outside of the scope of this PR, so perhaps we should add it to one of the weekly meetings for discussion of policy? We could also add the discussion to the mailing list. Your thoughts, Lana?

Reply:
I see this is addressed a little bit later, but we might want more clear guidelines. (And I think it would be good for us to clarify this just for ourselves anyhow.)

Reply:
Personally, I tend to think that the simpler the better with this kind of thing. Something like "at least two green ticks and no red crosses" (which I think is in the PR template already?).

Reply:
That sounds fair. We can also take this conversation online if needed, but I'm satisfied. I'll resolve this thread. 👍